### DISCUSSION TITLE
handling builder with 150 blocks?

### DISCUSSION THREAD

**Discussion by AbdelRahman-Elshafai** [filamentphp/filament#9710] [Help]
Package

filament/filament
Package Version

3.0.97
Laravel Version

10.32.1
Livewire Version

3.1.0
PHP Version

8.2.8

Problem description

i encountered an issue while adding blocks to my builder component that upon using alot of blocks it became really slow to update or add blocks?

**Comment by michaelanthonyfeather** [comment]
I am also having this issue with builders with less than 20 blocks
